src/MuTalk-Model/MethodMutation.class.st

```diff
@@ -137,6 +137,15 @@ MethodMutation >> runMutant [
 	] ensure: [ self uninstall ]
 ]
 
+{ #category : 'running' }
+MethodMutation >> runMutantWithoutStoppingOnError [
+
+	^ [ 
+		self install.
+		self runTestsWithoutStoppingOnError
+	] ensure: [ self uninstall ]
+]
+
```

Comment:
Here is the same, both this and  `runMutant` look the same except for the detail, maybe the boolean should arrive here?